## This file is to be automatically generated.


raise Exception("""Incorrectly imported file parameters.py. This file should be
                   generated before running the program. Do this now by running
                   casio\pre_process.py if you have not already done so.""")

